app/src/main/res/layout/weekday_item.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/custom_view"
+    android:orientation="horizontal"
+    android:layout_width="300dp"
+    android:layout_height="wrap_content"
+    android:background="@android:color/darker_gray"
+
+    android:gravity="center"
+    >
+
+    <Button
```

Comment:
We need to change these to ImageButtons so we can use Chloe's theme-colored weekday icons.